test/integration/targets/inventory_script/inventory.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cat inventory.json
```

Comment:
It would be helpful to have a comment explaining where this sample came from.

Reply:
addressed